packaging/RPMS/Fedora/rabbitmq-server.spec

```diff
@@ -203,9 +218,8 @@ systemctl try-restart %{name}.service >/dev/null 2>&1 || :
 %defattr(-,root,root,-)
 %attr(0755, rabbitmq, rabbitmq) %dir %{_localstatedir}/lib/rabbitmq
 %attr(0750, rabbitmq, rabbitmq) %dir %{_localstatedir}/lib/rabbitmq/mnesia
-%attr(0755, rabbitmq, rabbitmq) %dir %{_localstatedir}/log/rabbitmq
+%attr(0750, rabbitmq, rabbitmq) %dir %{_localstatedir}/log/rabbitmq
```

Comment:
Could line 219 be changed to `0750` instead, with line 221 deleted? I don't know the first thing about rpm packaging so excuse the question if it's obvious

Reply:
Line 219 is about another directory (`rabbitmq` user's home directory, `/var/lib/rabbitmq`), but perhaps I didn't understand your question?

Reply:
`lib` vs `log` ... maybe my screen font is too small 😸